src/test/java/ch/wisv/events/sales/controller/stats/SalesStatsControllersTest.java

```diff
@@ -0,0 +1,114 @@
+package ch.wisv.events.sales.controller.stats;
+
+import ch.wisv.events.ControllerTest;
+import ch.wisv.events.EventsApplicationTest;
+import ch.wisv.events.core.model.event.Event;
+import ch.wisv.events.core.service.event.EventService;
+import ch.wisv.events.sales.service.SalesService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+/**
+ * SalesScanMainControllerTest.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = EventsApplicationTest.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+public class SalesStatsControllersTest extends ControllerTest {
+
+    /**
+     * Mock of EventService
+     */
+    @MockBean
+    private EventService eventService;
+
+
+    /**
+     * Mock of salesService
+     */
+    @MockBean
+    private SalesService salesService;
+
+    /**
+     * Test stats event controller
+     *
+     * @throws Exception when exception
+     */
+    @Test
+    public void eventsControllerTest() throws Exception {
+
+        Event event = new Event(
+                "Test",
+                "test",
+                "test",
+                10,
+                10,
+                "path/to/files",
+                LocalDateTime.now(),
+                LocalDateTime.now().plusHours(1),
+                "Short description"
+        );
+
+        when(eventService.getByKey(event.getKey())).thenReturn(event);
```

Comment:
Make it an integration test by creating models and save them and removing the mocking (for most models a method in `ControllerTest` has been created). 
This test method is quite useless now.
In this cases your test cases do not test the added methods to the `OrderService`, `OrderProductRepo`, etc.

Reply:
Changed it to use the models found in `ControllerTest`. I will write tests for OrderService and OrderProductRepo next.